consts.go

```diff
@@ -24,8 +24,13 @@ const (
 )
 
 const (
-	metadataHeader  = "WAL"
-	metadataVersion = "1.0"
+	metadataHeader  = "github.com/NebulousLabs/writeaheadlog\n"
+	metadataVersion = "v1.0.0 \n" // Extra room to allow for versions like v3.1.41
+
+	// 48 bytes total, intentionally divisible by 16.
+	metadataHeaderSize  = 38
+	metadataVersionSize = 8
+	metadataStatusSize  = 2
```

Comment:
if the status size is 2 bytes, we do we treat it like a single byte when reading/writing?

Reply:
The first byte is the value, and the second byte is a newline. This should definitely be clarified.